README.md

```diff
@@ -1,3 +1,19 @@
-# cansw_processor_stm
+# cansw_processor_canards
 
-Software repo for the 2025 canards processor board.
+ Firmware for the 2024-2025 processor board project, performing state estimation and control for the canards system.
+ Project documentation can be found in the team Google Drive.
+
+## Developer Setup
+1. Clone repo locally
+2. Initialize git submodules
+   ```bash
+   git submodule update --init --recursive
+   ```
+4. Open project in STM32CubeIDE (recommended IDE)
+5. Use ST-Link to flash code to processor board
+
+## Project Structure
+- `STM32Cube/src/drivers/`: custom peripheral driver modules
+- `STM32Cube/src/application/`: high-level application logic modules
+- `STM32Cube/src/lib/`: third-party libraries
```

Comment:
IIRC when we had the kickoff meeting we were planning to have a different folder name for 3rd arty library flder :face: